### ISSUE TITLE
How to handle sub-amendments ?

### ISSUE THREAD

**Issue by RouxRC** [regardscitoyens/the-law-factory#144]
Currently they  are presented as regular amendments.
We should at least handle their specific case in the sidebar's title & content by mentioning so and adding a link that would focus on the parent amendment
Not sure whether we want to go further and visually dispay that info on the amendments blocks
What do you think @fmassot  @njoyard  ?

**Comment by njoyard** [comment]
I agree with sidebar changes, is that information already available in viz/amendements*.json ?

However I'm not sure how to display the relationship. Maybe a dotted line link to the parent amendment, but only shown when hovering ? I'm not a UI designer though... :)

Maybe we could add a visual clue of "grouped" amendments when using discussion order (see #145) with a common background or a small link betwen them (see examples below). The grouping may be used with sub-amendments as well as common discussion/identical amendments if we ever get that information.

![example](https://i.imgur.com/A5mC5Vy.png)

**Comment by RouxRC** [comment]
For the sidebar change, data is available in the json downloaded on each amendment (field sous_amendement_de )

For the viz, nope at the moment. One of your options could work, although that would break some of the sorting modes so I'm not sure it actually makes sense in the end.
